src/EventLoop/Driver.php

```diff
@@ -30,16 +30,15 @@ public function run(): void;
     public function stop(): void;
 
     /**
-     * Interrupts the event loop and continues with {main}.
+     * Create an object used to suspend and resume execution, either within a fiber or from {main}.
      *
-     * The driver MUST check for a set interrupt after invoking an event callback or microtask. If an interrupt exists,
-     * it must be reset, and the driver must suspend with the given callback, i.e. call \Fiber::suspend($callback);
+     * @param \Fiber $scheduler Fiber containing the running event loop.
      *
-     * @param callable $callback Callback to run on {main} before continuing.
+     * @return Suspension
      *
      * @internal This API is only supposed to be called by the Suspension API.
```

Comment:
This note is no longer relevant.

Reply:
My thinking was there's no reason for anyone to ever call this method themselves, but it doesn't have to be internal either.